pkg/tasks/introspect.go

```diff
@@ -40,7 +39,14 @@ func IntrospectHandler(ctx context.Context, task *models.TaskInfo, _ *queue.Queu
 		logger.Error().Err(errs[i]).Msgf("Error %v introspecting repository %v", i, p.Url)
 	}
 	logger.Debug().Msgf("IntrospectionUrl returned %d new packages", newRpms)
-	return nil
+	time.Sleep(time.Millisecond * 100)
```

Comment:
is this sleep needed?

Reply:
No...I'm not sure why this is here. I must have been testing something.